compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -45,7 +47,7 @@ class Functor f => Applicative f where
     -- | Lift a value.
     pure : a -> f a
 
-    -- | Sequential application.
+    -- | Sequentially apply the function.
```

Comment:
Not really sure what a better description would be but this seems like we should be able to come up with something better.

Reply:
Yeah, I really don't like this either, but I don't know how to improve.